build/assets/tuned/openshift-node-performance

```diff
@@ -42,6 +42,13 @@ net.core.busy_poll=50                         # network-latency
 net.ipv4.tcp_fastopen=3                       # network-latency
 vm.stat_interval = 10                         # cpu-partitioning  #realtime
 
+[irqbalance]
+{{if .GloballyDisableIrqLoadBalancing}}
+banned_cpus=${isolated_cores}
+{{else}}
+banned_cpus=""
```

Comment:
do we need to have this if GloballyDisableIrqLoadBalancing is false? meaning: could the following work?
```
{{if .GloballyDisableIrqLoadBalancing}}
[irqbalance]
banned_cpus=${isolated_cores}
{{end}}
```

Reply:
The parent tuned profile has:
```
[irqbalance]
banned_cpus=${isolated_cores}
```
I wanted to override that. [I will double check]

Reply:
OK, if you need to override it it's fine like it is.

Reply:
Now I am too (depends on the implementation of tuned). We will know in about 1 hour

Reply:
I would dig a bit more in https://github.com/openshift/cluster-node-tuning-operator/blob/master/assets/tuned/daemon/tuned/plugins/plugin_irqbalance.py to see the implications of an empty string and also see what you actually set on the system

Reply:
we had a test once checking IRQBALANCE_BANNED_CPUS value in the irqbalance config file when we had initramfs generated. maybe its time to bring this test back.

Reply:
@yanirq I actually tested live both cases and on empty string the cpu mask was 0 as expected
Regarding the mentioned test, I have something in mind, but  a cheat-sheet is always welcomed :) , do you remember the file or the test name? I will dig anyway.

Reply:
I've gone fast over the tuned code and it should be ok.
I will triple-check anyway, better safe...

Reply:
@fromanirh it took me a lot of time, but at least tuned started to work (MCO still can't install the rt kernel)
To answer your question, as I thought, if we don't add  banned_cpus="" the  isolated CPU set remains masked.
The reason is we inherit the cpu-partitioning bits, so we need to override banned_cpus with an empty value.
The other way around would work, e.g. setting banned_cpus only to empty string when needed, but I think a verbose mode is desired

Reply:
thanks for the extra check @marcel-apf . I'm fine with your approach. I just think this
```
f we don't add banned_cpus="" the isolated CPU set remains masked.
The reason is we inherit the cpu-partitioning bits, so we need to override banned_cpus with an empty value.
```
should be added as code comment for future reference.

Reply:
I changed the template to:
```{{if not .GloballyDisableIrqLoadBalancing}}
[irqbalance]
# Override the value set by cpu-partitioning with an empty one
banned_cpus=""
{{end}}
```
The reason is we don't know what exactly the parent use as banned cpus, they may update the formula in the future.
What we do know is we don't want to bann the CPUs for our case